cameraconnect/src/cameras.py

```diff
@@ -137,7 +140,7 @@ def _publish_mqtt(self, image) -> None:
         message = json.dumps(prepared_message)
         # Publish the message
         ret = self.client.publish(self.mqtt_topic, message, qos=0)
-        print(ret)
+        print("Image No.: " + str(ret[1]))
```

Comment:
Please use logging module for each use of print

Reply:
@JonaVo Please take a look at the logging (using the logging instead of print)

Reply:
done 1bc5eed